backend/.env.example

```
@@ -15,3 +15,7 @@ JWT_REFRESH_SECRET=your-super-secret-refresh-key-change-this-in-production
 
 # Frontend URL for CORS
 FRONTEND_URL=http://localhost:5173
+
+BCRYPT_ROUNDS=10
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The BCRYPT_ROUNDS key is duplicated

(DuplicatedKey)